services/httpd/privatebin.nix

```diff
@@ -0,0 +1,20 @@
+{ pkgs, ... }:
+with (import ./shared.nix);
+let
+  user = "wiki";
```

Comment:
lets not reuse the wiki user

Reply:
I will set up a proper user for the service later. I haven't quite decided if we want to add them to LDAP or hard code them into Nix. The latter is the standard procedure for nix.

Reply:
i think its better for our use case if we create service accounts, in ldap

Reply:
Should this be carried out before this PR is merged then?

Reply:
User added by Lucas, resolved.